docs/install_guides/installing-openvino-yocto.md

```diff
@@ -1,40 +1,86 @@
+
 This document provides instructions for creating a Yocto image with Intel® Distribution of OpenVINO™ toolkit.
 
 ## System Requirements
+
+@sphinxdirective
+.. tab:: Supported Linux Distributions
+
+   +--------------------------------+----------------------------------------+
+   | Operating System               | Version                                |
+   +================================+========================================+
+   | Ubuntu                         | 18.04 (LTS), 20.04 (LTS), 22.04 (LTS)  |
+   +--------------------------------+----------------------------------------+
+   | Fedora                         | 34, 35                                 |
+   +--------------------------------+----------------------------------------+
+   | AlmaLinux                      | 8.5                                    |
+   +--------------------------------+----------------------------------------+
+   | Debian GNU/Linux               | 10.x (Buster), 11.x (Bullseye)         |
+   +--------------------------------+----------------------------------------+
+   | OpenSUSE Leap                  | 15.3                                   |
+   +--------------------------------+----------------------------------------+
+
+.. tab::  Required Software
+
+   +---------------------------------------------+---------------------------+
+   | Software                                    | Version                   |
+   +=============================================+===========================+
+   | `GIT <https://git-scm.com/>`__              | 1.8.3.1 or greater        |
+   +---------------------------------------------+---------------------------+
+   | `tar <https://www.gnu.org/software/tar/>`__ | 1.28 or greater           |
+   +---------------------------------------------+---------------------------+
+   | `Python <https://www.python.org/>`__        | 3.6 or greater            |
+   +---------------------------------------------+---------------------------+
+   | `gcc <https://gcc.gnu.org/index.html>`__    | 7.5 or greater            |
+   +---------------------------------------------+---------------------------+
+
+@endsphinxdirective
+
+
 Use the [Yocto Project official documentation](https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html#compatible-linux-distribution) to set up and configure your host machine to be compatible with BitBake.
 
-## Step 1: Set Up Environment
+## Set Up Environment
 
-### Set Up Git Repositories
-The following Git repositories are required to build a Yocto image:
+### 1. Clone the repository.
 
-- [Poky](https://git.yoctoproject.org/poky)
-- [Meta-intel](https://git.yoctoproject.org/meta-intel/tree/README)
-- [Meta-openembedded](http://cgit.openembedded.org/meta-openembedded/tree/README)
-- <a href="https://github.com/kraj/meta-clang/blob/master/README.md">Meta-clang</a>
+```sh
+git clone https://git.yoctoproject.org/git/poky
```

Comment:
The kirkstone branch was added specifically in a PR earlier. I think we should add it here and in the following commands too.

Reply:
Right, will add the appropriate branches for the OpenVINO releases.